designer/submodules/packages/components/src/common/categoryChip.tsx

```diff
@@ -1,23 +1,56 @@
 import React, { useCallback, useMemo } from "react";
-import { Chip } from "@mui/material";
+import { Button, Chip, Typography } from "@mui/material";
 
 interface Props {
-    value: string;
-    filterValue: string[];
+    category: string;
+    filterValues: string[];
     setFilter: (value: string[]) => void;
 }
 
-export function CategoryChip({ value, filterValue, setFilter }: Props): JSX.Element {
-    const isSelected = useMemo(() => filterValue.includes(value), [filterValue, value]);
+export function CategoryChip({ category, filterValues, setFilter }: Props): JSX.Element {
+    const isSelected = useMemo(() => filterValues.includes(category), [filterValues, category]);
 
     const onClick = useCallback(
         (e) => {
-            setFilter(isSelected ? filterValue.filter((v) => v !== value) : [...filterValue, value]);
+            setFilter(isSelected ? filterValues.filter((v) => v !== category) : [...filterValues, category]);
             e.preventDefault();
             e.stopPropagation();
         },
-        [setFilter, isSelected, filterValue, value],
+        [setFilter, isSelected, filterValues, category],
     );
 
-    return <Chip tabIndex={0} label={value} size="small" color={isSelected ? "primary" : "default"} onClick={onClick} />;
+    return <Chip tabIndex={0} label={category} size="small" color={isSelected ? "primary" : "default"} onClick={onClick} />;
+}
+
+export function CategoryButton({ category, filterValues, setFilter }: Props): JSX.Element {
+    const isSelected = useMemo(() => filterValues.includes(category), [filterValues, category]);
+
+    const onClick = useCallback(
+        (e) => {
+            setFilter(isSelected ? filterValues.filter((v) => v !== category) : [...filterValues, category]);
+            e.preventDefault();
+            e.stopPropagation();
+        },
+        [setFilter, isSelected, filterValues, category],
+    );
+
+    return (
+        <Button
```

Comment:
You can use Button as a component in the Typography. You don't have to define fontSize and there won't be additional <p> Tag inside of the <button> component

```
<Typography
            component={Button}
            color={isSelected ? "primary" : "inherit"}
            sx={{
                textTransform: "capitalize",
                display: "flex",
                gap: 1,
                alignItems: "center",
                minWidth: "unset",
                p: 0,
                mx: 0,
            }}
            onClick={onClick}
            aria-selected={isSelected}
        >
            {category}
        </Typography>
```

Reply:
Thanks for the hint. Fixed.